index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Beryl's Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+
+</head>
+<body>
+<main>
+
+  <header>
+    <nav>
+      <ul class="logo" >
+        <li><a href=""> <h3 class="gakii"> Gakii Beryl</h3></a></li>
+        <li><a href=""><img class="menu" src="images/menu.svg" alt="Menu Icon" /></a></li>
```

Comment:
- [ ]Kindly put only the hamburger in the nav tag. Because of semantics reason, and the right position for convention and user experience for mobile devices